docs/_includes/sidebar-nav.html

```diff
@@ -10,16 +10,17 @@
         <div class="collapse {% for entry in item.items %}{% if page.url contains entry.url %} show {% endif %}{% endfor %}" id="{{ item.collapse-id }}-collapse">
         <ul class="btn-toggle-nav list-unstyled fw-normal pb-1 small">
           {% for entry in item.items %}
-            {% assign currentPage = entry.url | append: '/' %}
-            <li class="doc-link ps-3 pb-1 text-start {% if currentPage == page.url %} list-active {% endif %}">
+            {% assign currentEntry = entry.url | append: '/' %}
+            <li class="doc-link ps-3 pb-1 text-start {% if currentEntry == page.url %} list-active {% endif %}">
               <a href="{{ entry.url }}">
                 {{ entry.title }}
               </a>
-              {% if entry.subfolderitems[0] %}
-              <div class="collapse {% for subentry in entry.subfolderitems %}{% if page.url contains entry.url %} show {% endif %}{% endfor %}" id="{{ subentry.collapse-id }}-collapse">
-                <ul>
+              {% if entry.subfolderitems.size > 0 %}
+              <div class="collapse {% for subentry in entry.subfolderitems %}{% if page.url contains entry.url %} show {% break %}{% endif %}{% endfor %}" id="{{ subentry.collapse-id }}-collapse">
```

Comment:
What is the `id` attribute in this div used for? `subentry.collapse-id` doesn't exist in this context so it just outputs `<div id="-collapse" ... >`

Reply:
This can be removed. Were you planning to merge this right after my PR is merged?

Reply:
Ah was going to let you merge it into your PR. No matter, I'll rebase onto main and merge it there.